### DISCUSSION TITLE
Peacock colorbar is not showing correctly

### DISCUSSION THREAD

**Discussion by JingShuShi** [idaholab/moose#16611] [Q&A Tools] [answered]
Dear Moose Team,

I wanted to use moose for future simulations and now learning some basics.
When I visualize the result from example01 in peacock, my colorbar is fully black:
![colorbar](https://user-images.githubusercontent.com/22516734/103539993-4c2e5600-4e99-11eb-9f7e-923d88a74ae5.png)

How could I fix this problem?
Thanks for the help.

Regards,
Hao

**Comment by aeslaughter** [comment, marked as answer]
Are you using a Virtual Machine? If so, this is a known problem that we are unsure how to fix.

**Reply by JingShuShi** [reply]
Yes, I am using a VM on my Windows, thanks for the answer, I will need to reinstall moose on a proper ubuntu system.

**Reply by aeslaughter** [reply]
Paraview should work if you need something quick in the VM.

**Reply by JingShuShi** [reply]
Thanks, paraview would be a good temp solution for VM.
